istioctl/pkg/injector/injector-list.go

```diff
@@ -107,6 +107,15 @@ func injectorListCommand(ctx cli.Context) *cobra.Command {
 				return err
 			}
 			hooks := hooksList.Items
+
+			var istioHooks []admitv1.MutatingWebhookConfiguration
+			for _, hook := range hooks {
+				if hook.ObjectMeta.GetLabels()["app"] == "sidecar-injector" {
```

Comment:
I think we can add the label selector to List in L105

Reply:
Modified, thanks